packages/storage/src/database/global/metadata-database.ts

```diff
@@ -114,10 +114,14 @@ export class ModelMetadataDatabase extends ModelBasic<
         appPublicKey: string;
       */
       await collection.index({ databaseName: 1 }, { unique: true, session });
```

Comment:
I think we should allow unique database name per user, not per global

Reply:
Right now it global also. Or we can redesign it a prefix like `user_databaseName`

Reply:
unique index on (databaseOwner, databaseName) can solve this without having to prefix imo